### PR TITLE
Improved Institute Settings layout

### PR DESCRIPTION
**Pull request by northwestwitch**
Improves the readability and usage of Institute settings. I've realised that it was a bit messy and wanted to fix it a bit before adding the ClinVar settings as well

From this:

image

To this:

image

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

File: scout/server/blueprints/institutes/templates/overview/utils.html [hunk 1]
```diff
@@ -36,148 +36,188 @@
...
+                      {% endfor %}
+                    </select>
+                  </div>
+                </div>
+              </fieldset>
+            </div>
+            <!-- End of cohorts settings -->
 
-            <div>
+            <br>
```
Comment (dnil) [inline scout/server/blueprints/institutes/templates/overview/utils.html:134]: Consider a margin, padding or such. Like, the next row already has `mt-3` - perhaps increase it if you feel it is too small?

File: scout/server/blueprints/institutes/templates/overview/utils.html [hunk 2]
```diff
@@ -36,148 +36,188 @@
...
 
-              <div class="col-4">
-                {{ form.gene_panels.label(class="control-label",data_bs_toggle="tooltip", data_bs_placement="top", title="Select gene panels that will be available for variants filtering.") }}
-                <select multiple="multiple" name="gene_panels" class="selectpicker ml-3" data-style="btn-secondary">
-                  {% for choice in form.gene_panels.choices or [] %}
-                    <option value="{{choice[0]}}" {% if institute.gene_panels and choice[0] in institute.gene_panels.keys() %} selected {% endif %}>{{choice[1]}}</option>
-                  {% endfor %}
-                </select>
-              </div>
+            <br>
```
Comment (dnil) [inline scout/server/blueprints/institutes/templates/overview/utils.html:112]: 👀

File: scout/server/blueprints/institutes/templates/overview/utils.html [hunk 3]
```diff
@@ -36,148 +36,188 @@
...
+                        {% for instance in loqus_instances %}
+                          <option value="{{instance}}" {% if instance in institute.loqusdb_id %} selected {% endif %}>{{instance}}</option>
+                        {% endfor %}
+                      </select>
+                    </div>
+                  </div>
+              </div>
+              <!-- End of loqusdb settings -->
+
+              <br>
```
Comment (dnil) [inline scout/server/blueprints/institutes/templates/overview/utils.html:201]: As before! 😊

Comment (dnil) [reply scout/server/blueprints/institutes/templates/overview/utils.html:201]: Ah, or are the `row` plus explicit `d-flex` stuff causing some issue?

**Review by dnil — APPROVED** [review]
Looks great! Really nice with the theme grouping, and some very related stuff had really moved too far apart!

File: scout/server/blueprints/institutes/templates/overview/utils.html (hunk 3 above)
Comment (northwestwitch) [reply scout/server/blueprints/institutes/templates/overview/utils.html:201]: No issue, I just wanted to give it an extra space

Comment (dnil) [reply scout/server/blueprints/institutes/templates/overview/utils.html:201]: Ok! It's just layout so whatever, but looks cleaner with just a larger margin, like mb-4 or mb-5, doesn't it? BR is fine if you prefer it of course, but then perhaps drop the margins on the rows altogether?

Comment (northwestwitch) [reply scout/server/blueprints/institutes/templates/overview/utils.html]: No it's a good suggestion. I've replaced the newline with mt-5 and looks the same, thanks!

**Pull request merged**